src/runner/systemdconn.hpp

```diff
@@ -18,12 +18,30 @@
 
 namespace aos::sm::runner {
 
+/**
+ * Unit state.
+ */
+class UnitStateType {
+public:
+    enum class Enum { eFailed, eActive, eInterm };
```

Comment:
It you introduced enum for unit state, you can add all states:
https://www.freedesktop.org/software/systemd/man/latest/systemd.html